packages/manager/modules/bm-server-components/src/netboot/controller.js

```diff
@@ -0,0 +1,216 @@
+import forEach from 'lodash/forEach';
+import isFunction from 'lodash/isFunction';
+
+const regRootDevice1 = new RegExp('^/dev/[hs]d[ab][0-9]+$');
+const regRootDevice2 = new RegExp('^/dev/md[0-9]+$');
+const regRootDevice3 = new RegExp('^/dev/rd/c0p0[0-9]+$');
+const regRootDevice4 = new RegExp('^/dev/ram0$');
+
+export default class BmServerComponentsNetbootCtrl {
```

Comment:
Was this changed with a `git mv` or is this added code ?

Reply:
Yes moved with git mv. All controllers and templates are moved with git mv. I am not sure why git showing them as new files. May be because I have renamed them after moving?